opendatahub-io/odh-manifests
# Update ci test container to use centos:stream8 base image

This PR updates the CI tests base image to Centos Stream 8 since Centos is EOL.

This will resolve errors during setup of the CI environment due to `Error: Failed to download metadata for repo 'appstream': Cannot prepare internal mirrorlist: No URLs in mirrorlist` 

Test steps:
1. Deploy ODH with JupyterHub to allow a minimal amount of tests
1. Build the CI test  image `cd tests && make build`
1. Run the CI tests and confirm there are no errors`cd tests && make run SKIP_INSTALL=true TESTS_REGEX=jupyterhub` 

Signed-off-by: Landon LaSmith <LLaSmith@redhat.com>
